libraries/Microsoft.Bot.Builder.Ai/LocaleConverter.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Recognizers.Text;
+using Microsoft.Recognizers.Text.DateTime;
+
+namespace Microsoft.Bot.Builder.Ai
+{
+
+    //Struct used to store date format and time format for different locales
+    internal struct DateAndTimeLocaleFormat
+    {
+        private string timeFormat;
+        private string dateFormat;
+        public string TimeFormat
+        {
+            get
+            {
+                return timeFormat;
+            }
+            set
+            {
+                timeFormat = value;
+            }
+        }
+        public string DateFormat
+        {
+            get
+            {
+                return dateFormat;
+            }
+            set
+            {
+                dateFormat = value;
+            }
+        }
+
+    }
```

Comment:
consider auto-implemented properties:

internal struct DateAndTimeLocaleFormat
{
public TimeFormat {get; set; }
public DateFormat {get; set; }
}

Reply:
Yes I updated the code for auto-implemented properties